debian/libdaos-dev.install

```diff
@@ -3,5 +3,6 @@ usr/include/cart/*
 usr/include/gurt/*
 usr/lib64/libdaos.so
 usr/lib64/libcart.so
+usr/lib64/libcart_utils.so
```

Comment:
if we are installing a library, should we not also install the header file to use it?

Reply:
No longer a library but shared objectsd.